test/e2e/edu-1990n-helpers.js

```diff
@@ -0,0 +1,14 @@
+function completeServicePeriods(client, data) {
```

Comment:
Yeah, though there's some variation in the tours of duty fields now that might be hard to accomodate. I think I'll make a separate item for it.